connectors/src/connectors/microsoft/temporal/activities.ts

```diff
@@ -819,3 +977,67 @@ async function handleTextExtraction(
       }
     : null;
 }
+
+async function deleteFolder({
```

Comment:
We could probably move this function to a `utils.ts` file, wdyt?

Reply:
Yep good point, a `file.ts` like elsewhere
In another PR though to limit the rebase complexity of this one already big :sweat_smile:

Reply:
👌🏼